database/gorm/factory_test.go

```diff
@@ -0,0 +1,83 @@
+package gorm
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/goravel/framework/contracts/database/factory"
+	ormmocks "github.com/goravel/framework/contracts/database/orm/mocks"
+	"github.com/goravel/framework/database/orm"
+)
+
+type Person struct {
+	orm.Model
+	orm.SoftDeletes
+	Name   string
+	Avatar string
+}
+
+func (p *Person) Factory() factory.Factory {
+	return &PersonFactory{}
+}
+
+type PersonFactory struct {
+}
+
+func (p *PersonFactory) Definition() any {
+	faker := gofakeit.New(0)
+	return map[string]interface{}{
+		"name":       faker.Name(),
+		"avatar":     faker.Email(),
+		"created_at": faker.Date(),
+		"updated_at": faker.Date(),
+	}
+}
+
+type FactoryTestSuite struct {
+	suite.Suite
+	factory   *FactoryImpl
+	mockQuery *ormmocks.Query
+}
+
+func TestFactoryTestSuite(t *testing.T) {
+	suite.Run(t, new(FactoryTestSuite))
+}
+
+func (s *FactoryTestSuite) SetupTest() {
+	s.mockQuery = ormmocks.NewQuery(s.T())
+	s.factory = NewFactoryImpl(s.mockQuery)
+}
+
+func (s *FactoryTestSuite) TestTimes() {
```

Comment:
S: We need to test this method with the `Make` method, make sure the Person model is really `Make` multiple times.